acquisitions/src/main/resources/thunderjet/mod-consortia/features/tenant.feature

```diff
@@ -1,86 +1,174 @@
+  Scenario: Attempt to POST a second tenant with existing 'isCentral' = true or 'id' or 'name' or 'code' in this order
+    # cases for 409
+    # attempt to create a second central tenant ('isCentral' = true)
+    Given path 'consortia', consortiumId, 'tenants'
+    And request { id: '#(universityTenant)', code: 'XYZ', name: 'University tenants name', isCentral: true }
     When method POST
     Then status 409
-    And match response == {"errors":[{"message":"ERROR: duplicate key value violates unique constraint \"tenant_name_key\"\n  Detail: Key (name)=(test1) already exists.","type":"-1","code":"VALIDATION_ERROR"}]}
+    And match response == { errors : [{ message : 'Object with isCentral [true] is already presented in the system', type : '-1', code: 'DUPLICATE_ERROR' }] }
 
-    # Get Error when trying to save with a existed code
-    Given path 'consortia', '111841e3-e6fb-4191-8fd8-5674a5107c32', 'tenants'
-    And request { "id": "12345", "code": "ABD", "name": "test-new", "consortiumId": "111841e3-e6fb-4191-8fd8-5674a5107c32" }
+    # attempt to create a tenant with an existing id
+    Given path 'consortia', consortiumId, 'tenants'
+    And request { id: '#(centralTenant)', code: 'ABE', name: 'test1', isCentral: false }
     When method POST
     Then status 409
-    And match response == {"errors":[{"message":"ERROR: duplicate key value violates unique constraint \"tenant_code_key\"\n  Detail: Key (code)=(ABD) already exists.","type":"-1","code":"VALIDATION_ERROR"}]}
-
-    # Get Error when trying to update a tenant with a different id
-    Given path 'consortia', '111841e3-e6fb-4191-8fd8-5674a5107c32', 'tenants', '1234'
-    And request { "id": "12345", "code": "ABC", "name": "test", "consortiumId": "111841e3-e6fb-4191-8fd8-5674a5107c32" }
+    And match response.errors[0].message == 'Object with id [' + centralTenant +'] is already presented in the system'
+    And match response.errors[0].type == '-1'
+    And match response.errors[0].code == 'DUPLICATE_ERROR'
+
+#    # attempt to create a tenant with an existing code
+#    Given path 'consortia', consortiumId, 'tenants'
+#    And request { id: 'non-existing-tenant', code: 'ABD', name: 'test1', isCentral: false }
+#    When method POST
+#    Then status 409
+#    And match response == { errors : [{ message : 'ERROR: duplicate key value violates unique constraint \"tenant_code_key\"\n  Detail: Key (code)=(ABD) already exists.', type : '-1', code: 'VALIDATION_ERROR' }] }
+#
+#    # attempt to create a tenant with an existing name
+#    Given path 'consortia', consortiumId, 'tenants'
+#    And request { id: 'non-existing-tenant', code: 'ABE', name: 'Central tenants name updated', isCentral: false }
+#    When method POST
+#    Then status 409
+#    And match response == { errors : [{ message : 'ERROR: duplicate key value violates unique constraint \"tenant_name_key\"\n  Detail: Key (name)=(Central tenants name updated) already exists.', type : '-1', code: 'VALIDATION_ERROR' }] }
+
```

Comment:
tests will be uncommeted in scope of another story when issue with mod-authtoken will be resolved